components/tests/ui/testcases/web/post_test.txt

```diff
@@ -114,3 +200,203 @@ Test Chgrp
     Click Element                           id=launch_activities
     Wait Until Page Contains                Project moved to Group              60
 
+Test Drag And Drop Image
+
+    Select Experimenter
+    ${did}=                                     Create Dataset
+    Click Element                               refreshButton
+
+    Select Experimenter
+    ${pId}    ${datasetId}    ${imageId}        Select And Expand Project Dataset Image
+    Wait Until Element is Visible               xpath=//li[@data-id='${datasetId}']//li[@data-id='${imageId}']/a
+
+    Sleep                                       0.5
```

Comment:
Is there necessary to use sleep? There should be keyword guarantee tree element appeared.